contributors/guide/pull-requests.md

```diff
@@ -218,7 +218,7 @@ If you can extract whole ideas from your pull request and send those as pull req
 
 Kubernetes is a fast-moving codebase - lock in your changes ASAP with your small pull request, and make merges be someone else's problem.
 
-Multiple small pull requests are often better than multiple commits.
+Multiple small pull requests with fewer commits are often better than single pull requests with many commits.
```

Comment:
The updated statement makes it clear that one type of pull request is better than another type of pull request. Keen if anyone has thoughts on another way to make this distinction! :)